docs/application/native/guides/applications/component-based-app.md

```diff
@@ -0,0 +1,642 @@
+        };
+
+        return component_based_app_add_frame_component(comp_class,
+                component_id, &callback, user_data);
+    }
+    ```
+
+3.  Sending an action event to another application
+
+    Actions are declared in the manifest files. To send a proper action, you should carefully set app-control values according to the manifest file app-control specification. In this example, the app-control will be set for "dial-for-excel" action.
+    ```
+    static int __app_control_send(const char *app_id, const char *component_id,
+            void *user_data)
+    {
+        app_control_h handle = NULL;
+        int ret;
+
+        ret = app_control_create(&handle);
+        if (ret != APP_CONTROL_ERROR_NONE)
+            return ret;
+
+        ret = app_control_set_operation(handle, "http://tizen.org/appcontrol/operation/dial");
+        if (ret != APP_CONTROL_ERROR_NONE) {
+            app_control_destroy(handle);
+            return ret;
+        }
+
+        ret = app_control_set_mime(handle, "application/vnd.ms-excel");
+        if (ret != APP_CONTROL_ERROR_NONE) {
+            app_control_destroy(handle);
+            return ret;
+        }
+
+        ret = app_control_send_launch_request_async(handle,
+                __app_control_result_cb, NULL, user_data);
+        if (ret != APP_CONTROL_ERROR_NONE) {
+            app_control_destroy(handle);
+            return ret;
+        }
+        return 0;
+    }
+
+    static void __button_clicked_cb(void *data, Evas_Object *obj,
+        void *event_info) {
+        __app_control_send("org.tizen.base-component", "base-frame", NULL);
+    }
+    ```
+
+
+<a name="launch_application"></a>
+## Launch a Component-Based Application
+
+To launch a Component-Based Application:
+
+1. Declaring launch privilege in the manifest file
+    ```
+    <?xml version="1.0" encoding="utf-8"?>
+    <manifest xmlns="http://tizen.org/ns/packages" api-version="5.5" package="@PACKAGE_NAME@" install-location="internal-only" version="0.1.1">
+        <label>Sample</label>
+        <author email="tizenappfw@tizen.com" href="www.tizen.org">Tizen App Framework</author>
+        <description>Sample</description>
+        <ui-application appid="org.tizen.sample" exec="@BINDIR@/sample" nodisplay="false" multiple="false" type="capp" taskmanage="true" launch_mode="caller">
+            <label>Sample</label>
+            <icon>@DESKTOP_ICON@</icon>
+        </ui-application>
+        <privileges>
+            <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+        </privileges>
+    </manifest>
+    ```
+
+2.  Sending launch request
+
+    You can send launch request using app_control_h. The app_control_h will contains the Component-Based Application's ID and the component ID. The component ID is optional, if you do not set the component ID, then the main component instance will be created.
+
+    You can launch a Component-Based application as follow.
```

Comment:
To maintain consistency, please rephrase to:
You can launch a Component-Based application as shown in the following code:

Reply:
Done